exercises/02-greeting.py

```diff
@@ -1,9 +1,13 @@
 import sys
 
 def hello(name):
-    # TODO: Implement this correctly according to the exercise
-    print(name)
+    if name == "there":
+        print("Hello yourself!")
+    else:
+        reversed_name = ''.join(reversed(name))
+        print("Hello " + reversed_name + "!")
```

Comment:
This is good as it is, but another alternative is to use an "f-string", which would look like this:
```
print(f"Hello {reversed_name}!")
```
f-strings are a relatively new thing in Python